CONTRIBUTING.md

```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
 
 `$ gulp`
 
+#### Trouble shooting(For Mac)
+
+If you have problem serve the site locally after all these steps, please try the following steps:
+
+Launch Terminal (OS X) and [install RVM](https://rvm.io/rvm/install)
```

Comment:
Change "OS X" to "MacOS" - Apple recently rebranded the operating system.

These instructions do not appear to be limited to MacOS. Therefore, maybe we only need the link to the installation instructions...??